vehicles/src/modules/permit/dto/request/create-application.dto.ts

```diff
@@ -22,78 +31,107 @@ export class CreateApplicationDto {
     example: '',
     required: false,
   })
-  permitId: string;
+  @IsOptional()
+  @IsNumberString()
+  @MaxLength(20)
+  permitId?: string;
```

Comment:
So most of these changes to the dto (and other dtos) are just making some required fields optional right? I'm guessing there shouldn't be significant changes to the FE corresponding fields? or are there implications that the FE should be aware of?

Reply:
Yes, no changes to FE. We have updated dtos to place constraints wherever necessary.

Reply:
ok cool, thanks